5-AccessControl/2-call-api-groups/AppCreationScripts/sample.json

```diff
@@ -1,61 +1,64 @@
-        { "Comment": "This script has created a group named GroupAdmin for you. On Azure portal, assign some users to it, and configure your ID and Access token to emit GroupID in your app registration." },
-        { "Comment": "This script has created a group named GroupMember for you. On Azure portal, assign some users to it, and configure your ID and Access token to emit GroupID in your app registration." }
-      ]      
-    },
-    {
-      "Id": "client",
-      "Name": "msal-react-spa",
-      "Kind": "SinglePageApplication",
-      "Audience": "AzureADMyOrg",
-      "HomePage": "http://localhost:3000/",
-      "ReplyUrls": "http://localhost:3000/",
-      "GroupMembershipClaims": "SecurityGroup",
-      "RequiredResourcesAccess": [
-        {
-          "Resource": "service",
-          "DelegatedPermissions": [ "access_as_user" ]
+        "Id": "client",
+        "Name": "msal-react-app",
+        "Kind": "SinglePageApplication",
+        "Audience": "AzureADMyOrg",
+        "HomePage": "http://localhost:3000/",
+        "ReplyUrls": "http://localhost:3000/ , http://localhost:3000/redirect.html",
+        "GroupMembershipClaims": "SecurityGroup",
+        "PasswordCredentials": "Auto",
+        "scopes": ["access_via_groups_assignments"],
+        "Sample": {
+            "SampleSubPath": "5-AccessControl\\2-call-api-groups\\SPA",
+            "ProjectDirectory": "\\2-call-api-groups\\SPA"
         },
-        {
-          "Resource": "Microsoft Graph",
-          "DelegatedPermissions": [ "User.Read", "GroupMember.Read.All" ]
-        }
-      ],
-      "ManualSteps": [
-        { "Comment":  "Navigate to the portal and set the 'replyUrlsWithType' to 'Spa' in the application manifest" },
-        { "Comment": "This script has created a group named GroupAdmin for you. On Azure portal, assign some users to it, and configure your ID and Access token to emit GroupID in your app registration." },
-        { "Comment": "This script has created a group named GroupMember for you. On Azure portal, assign some users to it, and configure your ID and Access token to emit GroupID in your app registration." }
-      ]  
+        "SecurityGroups": [
+            {
+                "Name": "GroupAdmin",
+                "Description": "Admin Security Group"
+            },
+            {
+                "Name": "GroupMember",
+                "Description": "User Security Group"
+            }
+        ],
+        "RequiredResourcesAccess": [
+            {
+                "Resource": "client",
+                "DelegatedPermissions": ["access_via_groups_assignments"]
+            },
+            {
+                "Resource": "Microsoft Graph",
+                "DelegatedPermissions": ["User.Read", "GroupMember.Read.All"]
+            }
+        ],
+        "OptionalClaims": {
+            "IdTokenClaims": ["acct"]
+        },
+        "ManualSteps": [
+            {
```

Comment:
This should be removed: 
                
>                 Security groups matching the names you provided have been created in this tenant (if not present already). On Azure portal, assign some users to it, and configure ID & Access tokens to emit Group IDs"

 Instead, have these:
                
                { "Comment": "This script has created a group named GroupAdmin for you. On Azure portal, navigate to Azure AD > Groups blade and assign some users to it." },
                { "Comment": "This script has created a group named GroupMember for you. On Azure portal, navigate to Azure AD > Groups blade and assign some users to it." }